flatgeobuf/README.md

```diff
@@ -12,14 +12,14 @@ We aim to support any future version of FlatGeobuf, too.
 
 ## Collection
 
-The FlatGeobuf file must provide the [fiboa Collection](../core/README.md#collection) as a separate JSON file.
-Optionally, if the tool supports it, the fiboa collection can be written into the `Header` table's 
+The FlatGeobuf file must embed the [fiboa Collection](../core/README.md#collection) into the `Header` table's 
 [`metadata`](https://github.com/flatgeobuf/flatgeobuf/blob/ee7c8f5f45c67dd4a84a51fef518dfebc3e19d0a/src/fbs/header.fbs#L81) 
 field in a property named `fiboa` within a JSON-structured object.
+It is recommended to also provide the fiboa Collection as a separate JSON file.
 
 > [!NOTE]
-> As of GDAL/OGR 3.9.0 the [FlatGeobuf driver](https://gdal.org/drivers/vector/flatgeobuf.html)
-does not support writing the Header's `metadata` field. 
+> Starting in GDAL/OGR 3.9.0 the [FlatGeobuf driver](https://gdal.org/drivers/vector/flatgeobuf.html)
+supports writing to the Header's `metadata` field with the `-mo` option, i.e. `-mo "fiboa_version=0.2.0"`.
```

Comment:
Could it also write `{"fiboa": { "fiboa_version": "0.2.0", "fiboa_extensions": [] }}` for example?

Reply:
I'm still struggling to understand how the metadata is encoded within the header if you write with `-mo`. First I need an environment with GDAL 3.9. Also, I need to find out how to read the header without `ogrinfo` to really understand the encoding.

Reply:
No, not really. If we make this an official encoding I would add it to fiboa CLI at some point and you could use fiboa describe, but I haven't worked with FlatGeoBuf yet so not sure which tooling works best.

Reply:
I think this tutorial section will be useful: https://flatbuffers.dev/flatbuffers_guide_tutorial.html#autotoc_md190

Reply:
I documented an experiment here: https://github.com/fiboa/specification/issues/24#issuecomment-2161099119